extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -287,48 +293,163 @@ export class DataTierApplicationWizard {
 		}
 	}
 
+	// Cancel button on click event is using to send the data loss information to telemetry
+	private cancelDataTierWizard(): void {
+		if (this.model.potentialDataLoss) {
```

Comment:
For completeness it'd be easiest to send this event every time it's canceled so you can easily see how many cancels are done when there's possible data loss.